katas/valeria-gonzales/.gitignore

```diff
@@ -2,7 +2,8 @@
 ## files generated by popular Visual Studio add-ons.
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
-
+.gitignore
```

Comment:
Not sure how ignoring a .gitignore file works on a .gitignore file. A pretty philosophical question I would say. If there is any file or folder you want git to ignore, you should do it on the .gitignore file from the root of the project.